DESCRIPTION

```diff
@@ -47,13 +47,13 @@ LazyData: true
 Language: en-GB
 Depends: R (>= 3.1.0)
 Imports:
-    dust (>= 0.7.11),
+    dust (>= 0.7.9),
```

Comment:
Can you undo this change and make sure you compile the odin code into C++ with dust version 0.7.11?